src/components/MoneyRequestReportView/MoneyRequestReportTransactionList.tsx

```diff
@@ -306,7 +308,10 @@ function MoneyRequestReportTransactionList({
                                     disabled={isTransactionPendingDelete(transaction)}
                                 >
                                     <TransactionItemRow
-                                        transactionItem={transaction}
+                                        transactionItem={{
+                                            ...transaction,
+                                            violations: violations?.[transaction.transactionID] ?? [],
+                                        }}
```

Comment:
is there a way we could remove spreading in render? It always come with a performance cost